### ISSUE TITLE
EquivalentAddin element in manifest not removing Office Add-in UI from ribbon bar in Word or Excel on Windows

### ISSUE THREAD

**Issue by neville-jones** [OfficeDev/office-js#2621]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

# Provide required information needed to triage your issue

## Your Environment

* Platform: PC desktop
* Host: Excel and Word
* Office version number: 2203 (Build 15028.20248 Click-to-Run)
* Operating System: Windows 10
* Browser (if using Office on the web): 
 
## Expected behavior

We included the EquivalentAddins element in an Office web add-in's manifest; the EquivalentAddin child had the ProgId of a COM add-in with greater functionality than that of the web add-in. Based on the documentation https://docs.microsoft.com/en-us/office/dev/add-ins/develop/make-office-add-in-compatible-with-existing-com-add-in#centralized-deployment-of-office-add-in we expected the UI of the Office web add-in to be gone after Word or Excel was restarted because of the presence of the operating COM add-in.

## Current behavior

The button of the Office web add-in remained visible in the ribbon of Word and Excel after a restart. If the user clicked the button of the Office web add-in, the task pane was exposed on the right hand side, however, the content of the pane stayed a gray colour with the state of "Loading..." indefinitely shown.
So, the Office web add-in is not functional or useable, but the button of the Office web add-in has remained in the ribbon bar which was not expected (based on the documentation and experience with the EquivalentAddins element in Outlook).

## Steps to reproduce

To test this requires an Office web add-in and COM add-in, and a manifest of the Office web add-in with the correct EquivalentAddins element and children.
We cannot supply our add-ins as they are only commercially available.

## Link to live example(s)

N/A

# Provide additional details

## Context

End-users can become confused because there are two add-ins with similar names/functionality in the ribbon bar and only one is operational. It is good that only the COM add-in is operational, but the presence of the Office web add-in button in the toolbar leads to confusion and load on the internal help desk.

**Comment by pkkj** [comment]
Hi @neville-jones , would you like to provide more information (like screenshot or even a sample manifest) for this issue? Thanks.

**Comment by neville-jones** [comment]
(obfuscated) screenshot:

![image002](https://user-images.githubusercontent.com/62081302/173489303-b75875c7-0628-44f3-8b1f-c78ab963d7e4.png)

**Comment by ehsankalafchi** [comment]
We are experiencing almost the same behavior in Word. **One additional note**: I also noticed that it works as expected on the second window, so if you open Word you would see both AddIns in the ribbon side by side. But if you go to File menu and create/open another document, only one of the AddIns will be visible on the ribbon in the second Word window.

**Comment by oFlo193o** [comment]
Thanks for the Issue! I'm facing the same problem where the EquivalentAddin node does not remove the JS addin from the ribbon. 
@ehsankalafchi thanks for the hint of opening another Word file. I was able to reproduce this using Word, where the JS addin disappeared in the newly opened window.

I noticed that this "workaround" does not work for PowerPoint. In PowerPoint i still see both addins whereas Excel evaluated the EquivalentAddin node correctly.
